CHANGES.rst

```diff
@@ -1,3 +1,249 @@
+  in horizontal coordinates due to tracking a fixed ICRS coordinate.
+  Resulting in a pointing position written for each event, although the
+  resolution of the monitoring is much lower.
+  For simulated events, the event time is the timestamp of the simulation
+  and pointing is fixed in ``AltAz``.
+  ``ctapipe`` was using the closest point in time for simulated events when
+  reading data back in, however, this is problematic in case of many
+  simulation runs processed in parallel.
+
+  We now store the first received pointing information
+  in the ``configuration/telescope/pointing`` table per obs id,
+  only for simulation events. [`#2438 <https://github.com/cta-observatory/ctapipe/pull/2438>`__]
+
+- Replace ``n_signal`` and ``n_background`` options in ``ctapipe-train-particle-classifier``
+  with ``n_events`` and ``signal_fraction``, where ``signal_fraction`` = n_signal / (n_signal + n_background). [`#2465 <https://github.com/cta-observatory/ctapipe/pull/2465>`__]
+
+- Move the ``TableLoader`` options from being traitlets to
+  each ``read_...`` method allowing to load different data with the
+  same TableLoader-Instance.
+
+  In addition the default values for the options have changed. [`#2482 <https://github.com/cta-observatory/ctapipe/pull/2482>`__]
+
+- Adding monitoring: MonitoringCameraContainer as keyword argument to
+  the ``ImageCleaner`` API so cleaning algorithms can now access
+  relevant information for methods that e.g. require monitoring information. [`#2511 <https://github.com/cta-observatory/ctapipe/pull/2511>`__]
+
+- Unified the options for DataWriter and the data level names:
+
+  +-------------------------+--------------------------+
+  | Old                     | New                      |
+  +=========================+==========================+
+  | ``write_raw_waveforms`` | ``write_r0_waveforms``   |
+  +-------------------------+--------------------------+
+  | ``write_waveforms``     | ``write_r1_waveforms``   |
+  +-------------------------+--------------------------+
+  | ``write_images``        | ``write_dl1_images``     |
+  +-------------------------+--------------------------+
+  | ``write_parameters``    | ``write_dl1_parameters`` |
+  +-------------------------+--------------------------+
+  | ``write_showers``       | ``write_dl2``            |
+  +-------------------------+--------------------------+
+
+  This changes requires that existing configuration files are updated
+  if they use these parameters [`#2520 <https://github.com/cta-observatory/ctapipe/pull/2520>`__]
+
+
+Bug Fixes
+---------
+
+- Ensure that ``SubarrayDescription.reference_location`` is always generated by
+  ```SimTelEventSource``, even if the metadata is missing. In that case, construct a
+  dummy location with the correct observatory height and latitude and longitude
+  equal to zero ("Null Island").
+
+  From now one require that ``SubarrayDescriptions`` contain a reference_location
```

Comment:
This reads like a duplicate  of the above point, but looks like it was supposed to expand on it. Unfortunately I'm not sure what you were trying to add.

Reply:
The first point is regarding what the `SimTelEventSource` now does, the second point is a general requirement for the `SubarrayDescription`

Reply:
Ok, I think this would be clearer if the statements are in the reverse order, because the causal order, right?

As it is written I find the second paragraph to be mostly a confusing restatement of the first, because  "require that" there is a `reference_location` is not very different from "Ensure that" it "is always generated".

Reply:
Good point. I reformulated a bit and moved the general point to the "API changes" section